src/components/ValueTypeDropDown.vue

```diff
@@ -42,7 +42,7 @@ export default Vue.extend( {
 		optionItems(): PropertyValueRelationMenuItem[] {
 			return Object.values( PropertyValueRelation ).map( ( value: PropertyValueRelation ) => {
 				return {
-					label: value, // FIXME: replace with i18n -> T269453
+					label: this.$i18n( `query-builder-valuetyperelation-dropdown-${value}` ),
```

Comment:
please add all possible i18n keys here as comment, this would help in later finding orphan messages and finding out where an i18n message is used. 
See https://phabricator.wikimedia.org/T224429 and specially https://phabricator.wikimedia.org/T235502 for more context.

Reply:
good point. fixing!